app/code/Magento/CustomerGraphQl/Model/Customer/UpdateCustomerAccount.php

```diff
@@ -71,17 +78,19 @@ public function __construct(
         $this->dataObjectHelper = $dataObjectHelper;
         $this->restrictedKeys = $restrictedKeys;
         $this->changeSubscriptionStatus = $changeSubscriptionStatus;
+        $this->validateCustomerData = $validateCustomerData;
     }
 
     /**
-     * Update customer account data
+     * Update customer account
      *
      * @param CustomerInterface $customer
      * @param array $data
-     * @return void
      * @throws GraphQlAlreadyExistsException
      * @throws GraphQlAuthenticationException
      * @throws GraphQlInputException
+     * @throws \Magento\Framework\Exception\NoSuchEntityException
```

Comment:
Resolver should not throw ` \Magento\Framework\Exception\NoSuchEntityException` (just `\Magento\Framework\GraphQl\Exception\GraphQlNoSuchEntityException`)

Reply:
Fixed